app/models/alert.rb

```diff
@@ -118,6 +118,17 @@ def email_target_user
     update_attribute(:email_sent_at, Time.zone.now)
   end
 
+  def email_course_instructor
```

Comment:
I just mentioned this in chat, but since we'll need a new view and mailer anyway, I think the best approach for this will be put the email logic into the mailer rather than adding it to the Alert base class and also specifying both email targets in the monitor file.

Take a look at FirstStudentAlertManager, FirstEnrolledStudentAlert, and FirstEnrolledStudentAlertMailer, and FirstEnrolledStudentAlert as examples.

Doing it that way will mean we can continue to test the monitor in basically the same way we do now, but add a new isolated test (and preview) for the mailer to check that it is emailing all the right people.

Reply:
sure i will have a look at those files and try completing the spec

Reply:
@ragesoss @bwreid when i am trying to manually call methods for sending emails from rails console, it is giving some error `Mailgun::ParseError Exception: 785: unexpected token at 'Mailgun Magnificent API'` , i suspect thats because mail gun api key is not configured because i don't have .env file in my project; how can i set the api key and domain in my development.rb file ???